cfme/containers/provider/__init__.py

```diff
@@ -575,4 +575,20 @@ def navigate_and_get_rows(provider, obj, count, table_class=CheckboxTable,
     if not rows:
         return []
 
-    return sample(rows, min(count, len(rows)))
+    return random.sample(rows, min(count, len(rows)))
+
+
+# Chargeback header names: used in chargeback tests for convenience
+_HeaderNames = namedtuple('_HeaderNames', ['rate_name', 'metric_name', 'cost_name'])
```

Comment:
I like the idea here, but I don't think this is the right place for this data, as it doesn't actually seem to be related to provider implementation at all.

Same goes for `parse_number` below, that has nothing to do with providers. Check out some of the modules in util to see if it fits there (utils.units maybe??) or consider creating a new one.

Its likely these will be useful beyond containers.provider classes.